__init__.py

```diff
@@ -102,6 +158,22 @@ def handle_system_reboot(self, message):
     def handle_system_shutdown(self, message):
         subprocess.call(['/usr/bin/systemctl', 'poweroff'])
 
+    def handle_show_text(self, message):
+        self.log.debug("Drawing text to framebuffer")
+        text = message.data.get('text')
+        if text:
+            text = text.strip()
+            font = fit_font(text, self.find_resource(FONT_PATH, 'ui'), 30)
+            w, h = font.getsize(text)
+            image = Image.new('RGBA', (400, h), BACKGROUND)
+            draw = ImageDraw.Draw(image)
+            # Draw to center of screen
+            draw.text(((400 - w) / 2, 0), text,
+                      fill='white', font=font)
+            write_fb(image)
+            time.sleep(30)
```

Comment:
Our Mark 2 Pi enclosure will support time parameter in event? Or display_until_event (like mycroft.paired)?

Reply:
If we do something like the mark-2 enclosure there would be an idle override for any shown screen. either infinite or a set number of seconds.